src/libraries/System.Reflection/overview.md

```diff
@@ -0,0 +1,37 @@
+# Summary
+
+Reflection is used for late-bound introspection, invocation and IL code generation. It contains:
```

Comment:
I am not fan of duplicating content from official docs here. Can this contain just a link to the official docs and no other content? If the official docs do not have the right content, we should improve it there.

Reply:
I'll remove this overview.md file and instead just include an abbreviated description with a link the official docs.

I'll also add another .md file to contain the common descriptions for the "Status".